example-doc-win2019.md

```diff
@@ -0,0 +1,261 @@
+- Microsoft.VisualStudio.Workload.NetCrossPlat
+- Microsoft.VisualStudio.Workload.NetWeb
+- Microsoft.VisualStudio.Workload.Node
+- Microsoft.VisualStudio.Workload.Office
+- Microsoft.VisualStudio.Workload.Python
+- Microsoft.VisualStudio.Workload.Universal
+- Microsoft.VisualStudio.Workload.VisualStudioExtension
+
+#### Microsoft SSDT Visual Studio 2019 Extensions
+- Microsoft Analysis Services Projects Version: 2.9.8
+- SQL Server Integration Services Projects Version: 3.6
+- Microsoft Reporting Services Projects Version: 2.6.5
+
+#### Windows Driver Kit
+- WDK Version: 10.1.18362.1
+- WDK Visual Studio Extension Version: 10.0.18346.0
+
+#### WIX Tools
+- Toolset Version: 3.11.4516
+- WIX Toolset Studio 2019 Extension Version: 1.0.0.4
+
+
+### .NET Core
+- SDK: 2.1.300, 2.1.301,... , 3.1.200, 3.1.201
+- SDK Path: C:\Program Files\dotnet\sdk
+- Runtime: 2.1.0, 2.1.1, 2.1.10, 2.111,... , 3.1.2, 3.1.3
+- Runtime Path: C:\Program Files\dotnet\shared\Microsoft.NETCore.App
+
+### .NET Framework
+- Version: 4.8.03761
+- Path: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools
+
+### Package Management
+- Chocolatey 0.10.15
+- Vcpkg 2020.02.04
+- NPM 6.14.4
+- Yarn 1.22.4
+- Pip 20.0.2 (python 3.7.7)
+- Miniconda 4.6.14
+- RubyGems 3.1.2
+- Helm v3.2.0+ge11b7ce
+- Composer 1.10.5 2020-04-10 11:44:22
+
+### Project Management
+- Ant 1.10.5
+- Maven 3.6.3
+- Gradle 6.3
+
+### Tools
+- Docker 19.03.5
+- Docker-compose 1.25.4
+- Git: 2.26.2
+- Git LFS: 2.10.0
+- Hub CLI: 2.14.2
+- Subversion (SVN) 1.8.17
+- Packer 1.5.5
+- jq 1.6
+- MySQL 5.7.21.0
+- SQLServer PS 21.1.18221
+- SQLPS 1.0
+- MinGW 8.1.0
+- zstd 1.4.0
+- bazel 3.1.0
+- bazelisk v1.4.0
+- Azure CLI 2.5.0
+- AWS SAM CLI 0.48.0
+- AWS CLI 2.0.10
+- Kubectl v1.18.2
+- CMake 3.17.1
+- Azure CosmosDb Emulator 2.9.2.0
+- Azure DevOps Cli extension 0.18.0
+- Mercurial 5.0
+- zstd 1.4.0
+- Inno Setup 6.0.4
+- TLS12 1.2
+- Nullsoft Install System (NSIS) 3.5
+- Cloud Foundry CLI 6.51.0
+- VSWhere 2.8.4+ff0de50053 - shim 0.8.1
+- Kind v0.7.0
+- Alibaba Cloud CLI 3.0.26
+- WinAppDriver 1.1.1809.18001
+
+### Browsers
+- Google Chrome 81.0.4044.122
+- Microsoft Edge 81.0.416.64
+- Mozilla Firefox 75.0
+
+### Selenium Web Drivers
+- Chrome Driver 81.0.4044.69
+- Gecko Driver 0.26.0
+- IE Driver 3.8.0.0
+- Microsoft Edge Driver 81.0.416.64
+
+### Toolcache:
+#### Ruby (x64)
+- 2.4.10
+- 2.5.8
+- 2.6.6
+- 2.7.1
+
+#### Python (x86)(x64)
+- 2.7.18
+- 3.5.9
+- 3.6.10
+- 3.7.7
+- 3.8.2
+
+#### PyPy (x86)
+- 2.7.13
+- 3.6.9
+
+#### Boost
+- 1.69.0 [msvc-14.1]
+- 1.72.0 [msvc-14.2]
+
+### Azure Powershell modules
+| Module | Version | Path |
+|-|-|-|
+| Azure PowerShell module | 2.1.0<br>3.8.0<br>4.2.1<br>5.1.1<br>5.3.0 | C:\Modules\azure_2.1.0\Azure\2.1.0\Azure.psd1<br>C:\Modules\azure_3.8.0\Azure\3.8.0\Azure.psd1<br>C:\Modules\azure_4.2.1\Azure\4.2.1\Azure.psd1<br>C:\Modules\azure_5.1.1\Azure\5.1.1\Azure.psd1<br>C:\Modules\azure_5.3.0\Azure\5.3.0\Azure.psd1 |
```

Comment:
Do we need these paths? They are generic

Reply:
Reworked